docs/deployment_patterns/how_to_use_great_expectations_with_google_cloud_platform_and_bigquery.md

```diff
@@ -167,6 +167,12 @@ Then, load your DataContext into memory using the `get_context()` method.
 
 Next, load the following Datasource configuration that will connect to data in BigQuery,
 
+:::note
+
+Great Expectations previously asked users to define a named permanent table to be used as a "temporary" table with the name passed in as a `batch_spec_passthrough` parameter. This is no longer the case, and Great Expectations will automatically set tables that are created as the result of queries to expire after 1 day. 
+
```

Comment:
For additional context surrounding this change, please refer to the Discussion posted to our Github.?